llama_stack/distribution/ui/page/playground/tools.py

```diff
@@ -37,6 +37,15 @@ def reset_agent():
             label="Available ToolGroups", options=builtin_tools_list, selection_mode="multi", on_change=reset_agent
         )
 
+        if "builtin::rag" in toolgroup_selection:
+            vector_dbs = llama_stack_api.client.vector_dbs.list()
```

Comment:
```suggestion
            vector_dbs = llama_stack_api.client.vector_dbs.list() or []
            if not vector_dbs:
                st.info("No vector databases available for selection.")
```

How about this?

Reply:
Should I add this update to the RAG page as well? Its nearly the same code block. Maybe an opportunity to refactor some shared pieces in the playground in a later PR?

https://github.com/meta-llama/llama-stack/blob/b5a9ef4c6d9dd2a6d16383107bb9765da66a3faa/llama_stack/distribution/ui/page/playground/rag.py#L88-L96

Reply:
Later refactor sounds good.